### ISSUE TITLE
Create a project for iset.mm

### ISSUE THREAD

**Issue by savask** [metamath/set.mm#2738]
I suggest to create a github project "Intuitionising set.mm". There are many issues like "add X to iset.mm" while a project allows one to have all these goals in one place and add draft issues to the project without cluttering the main issues panel.

There's already a project for elementary geometry and I think it works well for tracking progress.

@jkingdon What do you think about the idea?

**Comment by jkingdon** [comment]
github projects are complicated enough (and apparently have changed since we made the elementary geometry one, which is now a "classic project") that I wasn't sure how to make one (or indeed whether it would be helpful or be cumbersome), but I don't really object.

**Comment by david-a-wheeler** [comment]
We certainly *could* create a separate repository for iset.mm, as compared to set.mm. I'm sceptical that we should.

Advantages of the current approach:

1. It's easier to synchronize iset.mm and set.mm. Not just their proofs, but also supporting scripts.
2. This allows us to break up the .mm files into multiple pieces and share some of the pieces. We aren't doing that today, but it has been actively discussed.

In short, I think there are mild advantages to the current combined repository, but if it's important to others, then let's do oit.

**Comment by savask** [comment]
@david-a-wheeler
I fully agree with what you wrote, but I meant a github project, not a separate repository (see the tab "Projects" at the top of the set.mm repo). I believe one doesn't need a separate repository for each project.

@jkingdon
"Classic projects" look simple enough and as far as I understand, we should be able to create one since there's already a classic project for elementary geometry. I don't see a button to create a project either, so maybe one has to be a member of the metamath organization to be able to do that?

**Comment by jkingdon** [comment]
I have created https://github.com/orgs/metamath/projects/1 . But it shows up as private - I don't think I have the permissions to make it public.

**Comment by digama0** [comment]
It should be public now.

**Comment by david-a-wheeler** [comment]
@savask - ah, thanks for the clarification!!

**Comment by jkingdon** [comment]
> It should be public now.

Thanks. Can you see it @savask ? I added the existing issues to it (all that belong there, unless I missed some).

**Comment by savask** [comment]
>Can you see it @savask ? I added the existing issues to it (all that belong there, unless I missed some).

Yes, I can see it!

I hope it will prove to be useful in organizing the iset.mm effort, especially that now one can form subgoals as draft issues. I think this issue can be closed now.
